### DISCUSSION TITLE
Unhide channel

### DISCUSSION THREAD

**Discussion by user119999** [retrospect-addon/plugin.video.retrospect#1656] [Q&A]
Hi,
I accidentally hit 'hide channel' in the channel menu inside the Retrospect addon and now I can't find a way to show the channel again. Anyone know how to get the channel back without reinstalling Retrospect?

BTW - is anyone using that feature?
Otherwise maybe @basrieter is ok with us starting a poll about him removing that feature?

Thanks in advance.

**Comment by basrieter** [comment]
Yes, can unhide them again via the Retrospect settings.

**Reply by basrieter** [reply]
![image](https://user-images.githubusercontent.com/1310443/201671467-97a54621-6b44-4d69-a15b-8dea609f4c91.png)

![image](https://user-images.githubusercontent.com/1310443/201671517-e243c96a-7820-41fd-9e8a-a7dcab5c93b5.png)

**Comment by basrieter** [comment]
Please continue that discussion here: https://github.com/retrospect-addon/plugin.video.retrospect/issues/1657
